src/components/NFTEditionSelectItemV2.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div
+    :class="[
+      'group',
+      'cursor-pointer',
+      'rounded-[16px]',
+      preset === 'dialog' ? 'w-full' : 'w-full desktop:w-[48%]',
+      'py-[18px] px-[24px]',
+      isSelected ? 'bg-[#E5F0F2]' : 'bg-gray-f7',
+      isSelected
+        ? 'border-[2px] border-like-green'
+        : 'border-shade-gray border-[1px] hover:border-[#d1d1d1] transition-all duration-75',
+      'flex items-center justify-between gap-[16px]',
+    ]"
+    @click="$emit('click', $event)"
+  >
+    <div class="flex items-center gap-[16px]">
+      <div class="relative">
+        <div
+          class="w-[20px] h-[20px] rounded-full bg-white border-shade-gray border-[1px]"
+        />
+        <IconCircleCheck
+          v-if="isSelected"
+          class="text-like-green w-[20px] absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2"
+        />
+      </div>
+      <div class="flex flex-col">
+        <div>
+          <NFTStockLabel :stock="stock" />
+        </div>
+        <div class="text-like-green text-[16px] font-500">{{ name }}</div>
+      </div>
+    </div>
+
+    <div class="flex flex-co whitespace-nowrap">
+      <div v-if="discountInfo" class="text-like-green text-[12px]">
+        {{ discountInfo.originalPriceLabel }}
+      </div>
+      <div class="text-like-green text-[16px] font-500">
+        {{ priceLabel }}
```

Comment:
Please use `v-text` for content

Reply:
[💄 Add discount original price label](https://github.com/likecoin/liker-land/pull/1948/commits/e8fdc2a56a0301fbd7904ec5cdb9f9cdd33c73a7)